Sources/SwiftInspectorVisitors/Tests/FileVisitorSpec.swift

```diff
@@ -131,12 +135,87 @@ final class FileVisitorSpec: QuickSpec {
           expect(matchingStructs.count) == 1
         }
 
+        it("finds TestClass.InnerEnum") {
+          let matchingStructs = self.sut.fileInfo.enums.filter {
+            $0.name == "InnerEnum"
+              && $0.parentTypeName == "TestClass"
+          }
+          expect(matchingStructs.count) == 1
+        }
+
+        it("finds TestEnum") {
+          let matchingStructs = self.sut.fileInfo.enums.filter {
```

Comment:
Nit: `matchingEnums`. It could be nice to make a name that is applicable to all types of tests to define away this class of copy/paste error.

Reply:
yeah I did that elsewhere. Good call on doing it everywhere.

ecbf472f8ca10b6d4e402bee535af689182d8071